tests/test_wave_03.py

```diff
@@ -54,13 +54,13 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
-
-    raise Exception("Test needs to be completed.")
+    
+    # raise Exception("Test needs to be completed.")
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
 
```

Comment:
Oop, looks like you commented out the Exception, but didn't add more assertions to this test. What assertions can you write to test that there are no duplicated movies in the list of movies watched only by friends?